architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,86 @@
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization determines whether an identity is allowed to perform any action.
+Note that authorization is a separate step from
+link:authentication.html[authentication], which is more about determining the
+identity of who is taking the action.
+
+Authorization is managed using:
+
+[horizontal]
+Rules:: Describe sets of permitted verbs on a set of resource types.
+Roles:: Collection of rules.
+Bindings:: Associate users and/or groups with a role.
+
+Users and groups may be bound to multiple roles at the same time.
+
```

Comment:
+1. I feel like these concepts (groups) could either be explained here, or in the architecture/core_objects/openshift_model section.